.github/workflows/build-test.yml

```diff
@@ -0,0 +1,59 @@
+name: Build Pipeline
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-electron:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
```

Comment:
The electron build is failing because its expecting node 22. You either need to adjust the code to set the requirement back to 20, which i think is done in the auth PR already inside of package.json or change this number to 22 instead of 20.

Reply:
updated to node version 22

Reply:
@funkel1989 also added the auth token within the repository settings under New repository secret